src/index.html

```diff
@@ -394,6 +400,25 @@ <h2 class="font-weight-bold">Contact</h2>
                 </div>
             </form>
         </section>
+
+        <section class="col-12 py-5" id="board-members">
+            <div class="text-center text-black mb-5">
+                <h2 class="font-weight-bold">Board Members</h2>
+                <ul style="list-style-type: none">
+                    <li> President: Elsbeth Magilton</a></li>
+                    <li> Vice President: Jenna Vitosh</a></li>
+                    <li> Treasurer & Finance Committee Chair: Bonita Sharif</a></li>
```

Comment:
I'm not sure we care to list financial responsibilities here (although I can be pursuaded). What's most important is officers vs non-officers. So listing

- President
- Vice President
- Secretary
- Treasurer
- At-Large Members

is probably sufficient.